packages/babel-parser/test/fixtures/flow/classes/constructor-override-with-class-prop-plugin/options.json

```diff
@@ -1,3 +1,4 @@
 {
+  "throws": "Classes may not have a field named 'constructor' (2:2)",
```

Comment:
It's correct :+1:

https://flow.org/try/#0MYGwhgzhAEBiD29oG8BQ1rHgOwgFwCcBXYPeAgLmgAoBKaAXgD5o8ALASwgG5UBfVEA